slack.md

```diff
@@ -1,147 +1,65 @@
-don't just say "Hi, can we talk?" or "Can we have a meeting later today?". Add some context and
-some plan of action:
-
-> Hey! I've been looking at the Automation Platform and I have a question. I'm wondering if directly
-  processes 1Tap receipts, or if they need to go through Receipt Bank for Accountants first. Do you
-  think we can have a meeting about that?
-
-This is much better! It doesn't just give the other person context about why you are contacting
-them – it also tells them what your question is. This gives them an opportunity to prepare a bit
-for the meeting, or even better – answer you right there in Slack and have no meeting at all
-(usually the best kind of meeting). Remember, the fewer interruptions we have, the more work we can
-get done.
-
-And that's all you need to know on messaging users.
+Avoid the cardinal sin of sending just “Hi” and waiting for the other side to reply. There is nothing wrong with pleasantries, but include why you are contacting the person and preferably something actionable for them to do. If you have a question and you think you might want to have a meeting, don't just say "Hi, can we talk?" or "Can we have a meeting later today?". Add some context and allow them to give you a meaningful reply.
 
 ## Statuses
 
-Please use statuses only as means to communicate availability with the other team members. The two
-most commonly used statuses should be ":palm_tree: Vacationing" and ":face_with_thermometer: Out
-Sick". Occasionally it might be good to set a custom status if you're working weird hours or harder
-to reach.
+The two most commonly used statuses should be "🌴 Vacationing" and “🤒 Out sick". 
 
-When you set your status to ":palm_tree: Vacationing", always edit the default text and say when
-you'll be back, e.g. "Vacationing (back on Sept 25th)". This will be very convenient for people who
-want to contact you as they'll immediately see when you'll be back. In addition to this, feel free
-to use Slack's feature to automatically remove the vacationing status on a certain date.
+Please use statuses only as means to communicate availability or if your current situation is atypical. If you are always working remotely, there is no point in setting "🏡 Working Remotely". Also, avoid setting status only as a joke.
 
-Set a status only if your current situation is untypical and you want to communicate this to the
-rest of the team. If you are always working remotely, there is no point in setting ":house: Working
-Remotely". Also, please avoid setting status only to be cute or funny – they should be used to
-communicate availability with the team, not express moods.
+When you set your status to "🌴 Vacationing", edit the default text to add when you'll be back, e.g. "Vacationing (back on Sept 25th)". This is very useful for people who want to contact you as they'll immediately see when you'll be back.
 
 ## Still got questions?
```

Comment:
How about we change this section and direct people to their managers instead of towards @skanev?

> If you have any questions about how we use Slack or you need help with setting something up, feel
free to ask in the #slack-guidelines channel.

Reply:
I've addressed the other comments in ninja-commits.

We can live with this ending for now. If @skanev says so, we can change this to something else.